foreshadow/core/data_preparer.py

```diff
@@ -0,0 +1,66 @@
+"""Data preparation and foreshadow pipeline."""
+from sklearn.pipeline import Pipeline
+
+from foreshadow.cleaners.data_cleaner import DataCleaner as _DataCleaner
+
+
+def _none_to_dict(**kwargs):
+    """Transform input kwarg to valid dict, handling sentinel value.
+
+    Accepts a single kwarg by default.
+
+    Args:
+        **kwargs: the kwarg value to ensure is proper format for kwargs.
```

Comment:
Any reason the input is **kwargs when it only takes a single input dict.

Reply:
I fixed this in the next PR -> realistically the two should really be combined